### PR TITLE
Develop to Production April 22, 2022

### PR DESCRIPTION
**Pull request by brandonmcastillo**
Changelog:
- updated all Gatsby packages to latest
- regenerated snapshots for tests

### CODE REVIEW

**Review by tristandenyer — APPROVED** [review]
LGTM

**Review by claireriley — APPROVED** [review]
LGTM!

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 1]
```diff
@@ -219,19 +219,20 @@ You can add documentation to your collection or requests to give it a summary, a
...
 
 ## Other Socket.IO notes
 
 Socket.IO typically uses WebSockets as its transport layer, but sometimes uses HTTP "long-polling" as a fallback when WebSockets can't be used. Postman doesn't support long-polling mode in Socket.IO.
 
 ## Troubleshooting WebSocket requests
 
-You can use the console to debug issues with a WebSocket connection. To open the console and view log messages, select **Console** in the status bar at the bottom left of Postman.
+You can use the console to debug issues with a WebSocket connection. To open the console and view log messages, select
+<img alt="Console icon" src="https://assets.postman.com/postman-docs/icon-console-v9.jpg#icon" width="16px"> **Console** in the Postman footer.
```
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:236]: Fix for this in https://github.com/postmanlabs/postman-docs/pull/4012

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 2]
```diff
@@ -96,13 +96,13 @@ To load a saved message:
...
 
-To rename, duplicate, or delete a saved message, select <img alt="Three dots icon" src="https://assets.postman.com/postman-docs/icon-three-dots-v9.jpg#icon" width="18px"> and then select **Rename**, **Duplicate**, or **Delete**.
+To rename, duplicate, or delete a saved message, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> and then select **Rename**, **Duplicate**, or **Delete**.
 
 ### Adding Socket.IO event names and arguments
 
 Sending events with a Socket.IO connection includes the ability to add event names and arguments. This makes it easy to listen to only specific events.
 
-For a Socket.IO connection, you can enter an event name to publish next to the **Send** button. If you select **Send** without entering a name, the default name `message` will be used.
+For a Socket.IO connection, you can enter an event name to publish next **Send**. If you select **Send** without entering a name, the default name `message` will be used.
```
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:105]: Fix for this in https://github.com/postmanlabs/postman-docs/pull/4012

**Review by HeatherStenson — APPROVED** [review]
Everything looks good except for the two errors noted in `/sending-requests/supported-api-frameworks/websocket.md`.

**Review by mark-anthony-postman — APPROVED** [review]

**Pull request merged**